scripts/train/yamls/finetune/gpt2-arc-easy--cpu-streaming-dataset.yaml

```diff
@@ -0,0 +1,77 @@
+max_seq_len: 512
+global_seed: 17
+
+# Run Name
+run_name:  # If left blank, will be read from env var $RUN_NAME
+
+# Model
+model:
+  name: hf_causal_lm
+  pretrained_model_name_or_path: gpt2
+  pretrained: true  # false: only use the architecture; true: initialize with pretrained weights
+
+# Tokenizer
+tokenizer:
+  name: gpt2
+  kwargs:
+    model_max_length: ${max_seq_len}
+
+# Dataloaders
+train_loader:
+  name: finetuning
+  dataset:
+    ############
+    remote: /Users/ning.wang/projects/llm-foundry/scripts/data_prep/example_data/
```

Comment:
should copy the structure of e.g. `gpt-neo-125m.yaml` in the `pretrain` folder for how to specify reasonable defaults for local/remote. Should not reference your specific path.